.gitignore

```diff
@@ -9,6 +9,8 @@
 # Ignore bundler config.
 /.bundle
 
+/.DS_store
+
```

Comment:
Some of these values can go in a global .gitignore file on your development machine.

It's not wrong to have them here, but as your projects grow numerous, each will need some of these "givens".

Here's someone's explanation with commands and notes: 
https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
I didn't know about this, thanks for letting me know.